conda/bootstrap.py

```diff
@@ -305,7 +305,7 @@ def build_spack_env(config, update_spack, machine, compiler, mpi, env_name,
     if esmf != 'None':
         specs.append(f'esmf@{esmf}+mpi+netcdf~pio+pnetcdf')
     if scorpio != 'None':
-        specs.append(f'scorpio@{scorpio}+pnetcdf~timing+internal-timing~tools+malloc')
+        specs.append(f'scorpio@{scorpio}+pnetcdf+timing+internal-timing~tools+malloc')
```

Comment:
I'm turning on timing for now.  In future scorpio versions, we can decide if we want it turned off but it's required in this one because of a known bug (https://github.com/E3SM-Project/scorpio/issues/476)